homeassistant/components/gardena/__init__.py

```diff
@@ -0,0 +1,74 @@
+"""Support for Gardena Smart sytem devices."""
+import logging
+import voluptuous as vol
+from gardena.smart_system import SmartSystem
```

Comment:
Already mentioned that import should be grouped and sorted.

Reply:
Do you have a pointer to the documentation ? 
Let me know if my modifications are fine.